src/Components/Link/index.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+import './index.css'
+
+
+export default class ApplyLink extends Component {
+    render() {
+        return (
+            <a className="Btn Btn-Apply" href={this.props.LinkUrl} > <span>{this.props.LinkText}</span> </a>
```

Comment:
I would change `this.props.LinkText` to `this.props.children`. `children` is a special kind of props that allows us to pass complex html structures into the component. Please take a look [here](https://codeburst.io/a-quick-intro-to-reacts-props-children-cb3d2fce4891)